src/chi-vue/public/index.html

```diff
@@ -5,10 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>Chi Vue</title>
-    <link rel="stylesheet" href="https://assets.ctl.io/chi/5.0.0/chi.css">
-    <link rel="icon" type="image/svg+xml" href="https://assets.ctl.io/chi/5.0.0/assets/images/favicon.svg">
-    <link rel="alternate icon" href="https://assets.ctl.io/chi/5.0.0/assets/images/favicon.ico">
-    <script src="https://assets.ctl.io/chi/5.0.0/js/chi.js"></script>
+    <link rel="stylesheet" href="http://localhost:8000/chi-portal.css">
+    <link rel="icon" type="image/svg+xml" href="http://localhost:8000/assets/images/favicon.svg">
+    <link rel="alternate icon" href="http://localhost:8000/assets/images/favicon.ico">
+    <script src="http://localhost:8000/js/chi.js"></script>
```

Comment:
Is this correct?

Reply:
Yes, Chi assets on local are served from these URLs.
This is not a production distributed code, just a development one.

<img width="460" alt="Screenshot 2021-08-02 at 15 10 02" src="https://user-images.githubusercontent.com/25136131/127867306-891c96ec-0a1e-4a51-8830-27f20396e697.png">